var/spack/repos/builtin/packages/py-requests/package.py

```diff
@@ -33,3 +35,4 @@ class PyRequests(PythonPackage):
     depends_on('py-urllib3@1.21.1:1.26', type=('build', 'run'), when='@2.25.0:')
     depends_on('py-urllib3@1.21.1:1.24,1.25.2:1.25', type=('build', 'run'), when='@2.16.0:2.24')
     depends_on('py-certifi@2017.4.17:', type=('build', 'run'), when='@2.16.0:')
+    depends_on('py-pysocks@1.5.6,1.5.8:1', type=('build', 'run'), when='+socks')
```

Comment:
```suggestion
    depends_on('py-pysocks@1.5.6,1.5.8:', type=('build', 'run'), when='+socks')
```
Is there actually an upper bound here?

Reply:
[Nope](https://github.com/psf/requests/blob/9dd6e47b5b4d25ee34701a1a0111a3332b1478d7/setup.py#L106), I got inspiration from what is defined in the `urllib3` file.